package newjavalearn;

public class Exceptions {

	public static void main(String[] args) {

try
{
	System.out.println(10/0);
	
}


catch(ArithmeticException e)
{
	System.out.println("last statment");

}
finally
{
	System.out.println("finallllllll");
}



	}

	}


